dspace-api/src/main/java/org/dspace/app/util/AuthorizeUtil.java

```diff
@@ -410,15 +414,15 @@ else if (!AuthorizeManager.isAdmin(context))
     public static void authorizeRemoveAdminGroup(Context context,
             Collection collection) throws AuthorizeException, SQLException
     {
-        Community[] parentCommunities = collection.getCommunities();
+        List<Community> parentCommunities = collection.getCommunities();
         if (AuthorizeConfiguration
                 .canCommunityAdminManageCollectionAdminGroup()
-                && parentCommunities != null && parentCommunities.length > 0)
+                && parentCommunities != null && parentCommunities.size() > 0)
         {
-            AuthorizeManager.authorizeAction(context, collection
-                    .getCommunities()[0], Constants.ADMIN);
+            authorizeService.authorizeAction(context, collection
+                    .getCommunities().get(0), Constants.ADMIN);
```

Comment:
Is this the proper Service way, or needs another pass?
collection.getCommunities().get(0)

I can't find collectionService.getCommunities(collection), do I need to create it? I see getParentObject